jabgui/src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -383,6 +390,37 @@ private SplitPane getPaneAndStartSearch(BibEntry entry) {
         refreshCitedByButton.setTooltip(new Tooltip(Localization.lang("Restart search")));
         styleTopBarNode(refreshCitedByButton, 15.0);
 
+        // Dropdown for citationFetcher
+        ComboBox<CitationFetcherType> fetcherCombo = new ComboBox<>(
+                FXCollections.observableArrayList(
+                        CitationFetcherType.SEMANTICSCHOLAR,
+                        CitationFetcherType.CROSSREF
+                )
+        );
+        fetcherCombo.setTooltip(new Tooltip(Localization.lang("Select Citation Fetcher")));
+        fetcherCombo.setPrefWidth(160);
+        styleTopBarNode(fetcherCombo, 75.0);
+        fetcherCombo.setConverter(new StringConverter<CitationFetcherType>() {
+            @Override
+            public String toString(CitationFetcherType citationFetcherType) {
+                return citationFetcherType.getName();
+            }
+
+            @Override
+            public CitationFetcherType fromString(String s) {
```

Comment:
1. Add a comment when the variable can be null
2. Add `@Nullable` to the parameter